codeforboston/maple
# About Copy Updates

# Summary

issue #1604 

change copy and update logo where requested

1. About -> Team -> Partners -> Partners in Democracy
2. About -> Support Maple -> Donate
3. About -> FAQ -> The MAPLE Team + Getting Involved

added translation support for About -> Team -> Partners -> Page Descr

# Checklist

- [n/a] If I've added shared components, I've added a storybook story.
- [n/a] I've made pages responsive and look good on mobile.

# Screenshots

![image](https://github.com/user-attachments/assets/1586c0c4-34c5-408d-b596-b769caeddd69)

![image](https://github.com/user-attachments/assets/e491d12a-c11b-441d-a1b7-6995d3caf257)

![image](https://github.com/user-attachments/assets/794e2574-9e68-466e-998c-a66de54df52a)

# Known issues

N/A

# Steps to test/reproduce

1. Go to the relevant subpages
2. Check that the requested copy is updated

-
